src/sql/workbench/parts/dataExplorer/browser/dataExplorer.contribution.ts

```diff
@@ -88,3 +94,95 @@ configurationRegistry.registerConfiguration({
 		}
 	}
 });
+
+export interface IDataExplorerActionContribution {
+	category: string;
+	commandId: string;
+	label: string;
+	icon?: {
+		light: string;
+		dark: string;
+	};
+	isPrimary: boolean;
+}
+
+const DataExplorerActionSchema: IJSONSchema = {
+	type: 'object',
+	properties: {
+		category: {
+			type: 'string',
+			description: localize('azdata.extension.contributes.dataExplorer.action.category', "Category of the action used for grouping the actions")
+		},
+		commandId: {
+			type: 'string',
+			description: localize('azdata.extension.contributes.dataExplorer.action.commandId', "Id of the command that will be executed")
+		},
+		label: {
+			type: 'string',
+			description: localize('azdata.extension.contributes.dataExplorer.action.label', "Display name of the action")
+		},
+		icon: {
+			type: 'object',
+			description: localize('azdata.extension.contributes.dataExplorer.action.icon', "Icon for the action")
```

Comment:
Give this a properties sub-element as well describing the light and dark properties?